tools/SRS_tools/alpha_beta.xml

```diff
@@ -0,0 +1,177 @@
+<tool id="SRS_diversity_maps" name="Map diversity" version="@VERSION@" profile = "20.01">
+    <description>from  remote sensing data</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="SRS_requirements">
+        <requirement type="package" version="0.62.0">r-matrixStats</requirement> 
+        <requirement type="package" version="0.51.2">r-rastervis</requirement>
+        <requirement type="package" version="1.25.0">r-future</requirement>
+        <requirement type="package" version="1.9.0">r-future.apply</requirement>
+        <requirement type="package" version="3.3.6">r-ggplot2</requirement>
+        <requirement type="package" version="0.31">r-xfun</requirement>
+        <requirement type="package" version="2.11.0">r-mapview</requirement>
+        <requirement type="package" version="0.1.0">r-leafpop</requirement>
+        <requirement type="package" version="1.1_3">r-rcolorbrewer</requirement>
+        <requirement type="package" version="2.0_1">r-labdsv</requirement>
+        <requirement type="package" version="2.3">r-gridextra</requirement>
+        <requirement type="package" version="1.14.2">r-data.table</requirement>
+        <requirement type="package" version="2.0.9">r-ecodist</requirement>
+        <requirement type="package" version="13.3">r-fields</requirement>
+        <requirement type="package" version="0.4.6">r-geometry</requirement>
+        <requirement type="package" version="1.0.2">r-matlab</requirement>
+        <requirement type="package" version="0.4_4">r-snow</requirement>
+        <requirement type="package" version="2.6_2">r-vegan</requirement>
+        <requirement type="package" version="2.4.2">r-remotes</requirement> 
+        <requirement type="package" version="3.0.0">r-emstreer</requirement>
+        <requirement type="package" version="1.6.1">r-mmand</requirement>
+        <requirement type="package" version="1.0">r-dissutils</requirement>  
+        <requirement type="package" version="3.2.2">r-filesstrings</requirement>  
+    </expand>
+    <command detect_errors="exit_code"><![CDATA[ 
+        #if $method.origin == 'envi_bil': 
+          cp '$input_raster' '${input_raster.element_identifier}' &&
```

Comment:
Not sure what you want to do here. But you are copying to an input file, that is not allowed and will overwrite your inputs.

Reply:
I see, but the element_identifier can have wired characters I think. If that is a problem try to clean it, like here: 

https://github.com/galaxyproject/tools-iuc/blob/c38fd63f7980c70390d104a73ba4c72b266444c3/tools/vsnp/vsnp_statistics.xml#L11

Also you might want to use `ln` instead of `cp`, if this is possible.